gensim/models/keyedvectors.py

```diff
@@ -1603,13 +1602,14 @@ def word_vec(self, word, use_norm=False):
             # from gensim.models.fasttext import compute_ngrams
             word_vec = np.zeros(self.vectors_ngrams.shape[1], dtype=np.float32)
             ngrams = _compute_ngrams(word, self.min_n, self.max_n)
-            ngrams = [ng for ng in ngrams if ng in self.ngrams]
             if use_norm:
                 ngram_weights = self.vectors_ngrams_norm
             else:
                 ngram_weights = self.vectors_ngrams
             for ngram in ngrams:
-                word_vec += ngram_weights[self.ngrams[ngram]]
+                ngram_hash = _ft_hash(ngram) % self.bucket
+                if ngram_hash in self.hash2index:
+                    word_vec += ngram_weights[self.hash2index[ngram_hash]]
             if word_vec.any():
                 return word_vec / len(ngrams)
```

Comment:
This probably needs to be updated to only take into account the ngrams for which hashes were present in `self.hash2index`